lunes_cms/cms/admins/document_admin.py

```diff
@@ -129,6 +130,26 @@ def related_training_set(self, obj):
 
     related_training_set.short_description = _("training set")
 
+    def related_disciplines(self, obj):
```

Comment:
To be honest, I don't really like obj as a variable name here. I'd rather call it something like documents. However I know we use obj quite often in Lunes and PEP8 doesn't write anything about whether to use or avoid Hungarian style, so I'll leave it up to you to decide whether consistency or a single better named parameter is more important here :sweat_smile:

Reply:
I thought "obj" is not a cool namimg too 😅 I'll leave it as it is for now and let's rename them all in another issue/PR 😸